src/assets/styles/variables.css

```diff
@@ -29,6 +29,8 @@
   --color-tan-darker: rgb(121, 87, 47);
   --color-princeton-salmon: rgb(255, 201, 148);
   --color-princeton-orange-on-black: rgb(245, 128, 37);
+  --color-princeton-orange-50: #E77500;
```

Comment:
colors from the [WDS styleguide](https://drive.google.com/file/d/1AgLHDP4WtKLHVWLtex7X0E7mgj97Cx0P/view?usp=sharing)